locale/de.yml

```diff
@@ -2,7 +2,7 @@ de:
   product:
     name: ManageIQ
     name_full: ManageIQ
-    copyright: "Copyright (c) 2021 ManageIQ. Sponsored by Red Hat Inc."
+    copyright: "Copyright (c) 2022 ManageIQ. Sponsored by Red Hat Inc."
```

Comment:
I assume we'll want to update the "Sponsored by Red Hat Inc." part as well @Fryguy

Reply:
Red Hat still owns the domain and other stuff.